examples/light-switch-app/light-switch-common/BUILD.gn

```diff
@@ -14,8 +14,13 @@
 
 import("//build_overrides/chip.gni")
 import("${chip_root}/src/app/chip_data_model.gni")
+import("${chip_root}/src/app/icd/icd.gni")
 
 chip_data_model("light-switch-common") {
-  zap_file = "light-switch-app.zap"
+  if (chip_enable_icd_lit) {
+    zap_file = "lit-icd-light-switch-app.zap"
```

Comment:
should the name order be consistent? one is lit-icd and the config is icd_lit

Reply:
done